src/utils/DateTimeUtils.js

```diff
@@ -13,9 +13,26 @@ export const areDatesSame = (date1, date2) => {
   return formatDate(date1) === formatDate(date2);
 };
 
-export const epochTo24HourTimeFormat = (epochSeconds) => {
+export const epochTo24HourTimeFormat = (epochSeconds, includeDate = false) => {
   const formattedTime = moment.unix(epochSeconds).format("HH:mm");
-  return formattedTime;
+
+  if (formattedTime === "00:00" && includeDate) {
+    const currentDate = moment.unix(epochSeconds).format("DD/MM/YYYY");
```

Comment:
please pick the format from existing constant

Reply:
Thanks, Its refactored now